embedded/ahtree/ahtree.go

```diff
@@ -775,7 +768,17 @@ func (t *AHtree) sync() error {
 		return nil
 	}
 
-	err := t.pLog.Sync()
+	err := t.pLog.Flush()
```

Comment:
WDYT of combining flush and sync into another method on the appendable interface? Like `FSync` or anything else? Will just make it easier to use in future cases where a person wants to commit, preserving durability

Reply:
I though about even removing Flush method from Appendable interface, however it would require further changes and that's why I keep it as it was, at least in the scope of this changes

Reply:
ack